samples/xamarin-forms/Cinephile/Cinephile/Views/UpcomingMoviesListView.xaml

```diff
@@ -6,11 +6,15 @@
                     xmlns:ui="clr-namespace:Cinephile.Views"
                     xmlns:vms="clr-namespace:Cinephile.ViewModels"
                     x:TypeArguments="vms:UpcomingMoviesListViewModel">
-
+                    
```

Comment:
Warning here! We have now a separated project for ViewModels and you *MUST* specify the assembly `assembly=Cinephile.ViewModels`. It would look like below:

```
xmlns:vms="clr-namespace:Cinephile.ViewModels;assembly=Cinephile.ViewModels"
```